src/domain/synthetics/trade/usePriceImpactWarningState.ts

```diff
@@ -10,12 +10,12 @@ import { TradeFlags } from "./types";
 export type PriceImpactWarningState = ReturnType<typeof usePriceImpactWarningState>;
 
 export function usePriceImpactWarningState({
-  positionPriceImpact,
+  positionCollateralPriceImpact,
```

Comment:
Why do we renaming this option but didn't change naming of returned fields and internal variabled like `isHighPositionImpact` or `isHighPositionImpactAccepted`?

Reply:
Good point, changed back the variable naming. I think it's better to keep the usePriceImpactWarningState hook more abstract. So it doesn't matter which kind of priceImpact it receive.